### ISSUE TITLE
"Availability", not "accessibility"

### ISSUE THREAD

**Issue by fredrika11y** [w3c/fingerprinting-guidance#48]
In a few places, the document mentions the word "accessibility" in a context where "availability" might just as well have been used. In current discourse, "accessibility" usually refers to themes concerning access for persons with disabilities. There wording runs no risk of resulting in misconstructions with implications for this field; however, the fact that the word is used here to mean something, namely "availability", which it is currently not usually signifying, may be confusing to a reader. Unless referring specifically to themes pertaining to (persons with) disabilities and the discourse on this subject, please consider refraining from the use of "accessible" and related words and instead look to "available" etc. Thank you.

**Comment by tomrittervg** [comment]
I think this is addressed by now?

**Comment by npdoty** [comment]
Yes, I believe this is fixed. We still occasionally use the word "access" as in "access to" but that's less likely to be confusing in this way. Closing for now, but accessibility folks should re-open or open other issues if there is other phrasing with a potential conflict.
